pages/home/auth-providers/oauth2.mdx

```diff
@@ -96,9 +96,9 @@ The only supported OAuth 2.0 flow is the authorization code grant (with or witho
 This section configures the initial authorization request.
 
 - `endpoint`: The authorization endpoint for your OAuth 2.0 server, e.g. `/oauth/authorize`
-- `params`: A map of parameter keys and values to include in the authorization request.
+- `parameters`: A map of parameter keys and values to include in the authorization request.
```

Comment:
Validation caught that it's `parameters` not `params`. Turns out this was always wrong in the docs 🤦‍♂️

Reply:
I think it should be `params` actually: https://github.com/ArcadeAI/Engine/blob/fc76ae1a08792e3635a818b9c6068436308d4862/pkg/auth/config.go#L107

Reply:
Thank you for catching this 🙏  You are right!

I got distracted by: https://github.com/ArcadeAI/Engine/blob/main/pkg/auth/oauth2.go#L123

Since `params` already works, I will correct the docs. I'll also change that ^ to be `params` for consistency, unless you have any objections.

Reply:
works for me! But we'll need to rev the versioned jsonb if you change that

Reply:
Just reverted this doc to `params`